### DISCUSSION TITLE
What is best way to get github badges quickly?

### DISCUSSION THREAD

**Discussion by Sun-Shine2** [community/community#21170] [Other Feature Feedback, Questions, & Ideas] [answered]
I am going to get several github badges. But I don't know how to do it. Please let me know this.

**Comment by radar-blip** [comment]
Here is info on badges and how to get them: https://github.com/drknzz/GitHub-Achievements

**Comment by patronus5** [comment, marked as answer]
https://github.com/Schweinepriester/github-profile-achievements

You can refer to the URL to earn several badges.

**Comment by whoami-anoint** [comment]
https://github.com/Schweinepriester/github-profile-achievements

You can refer to the URL to earn several badges.

**Comment by NitinIsFit** [comment]
Pull request your own repos and close your own issues in 5 minutes, i got most of them like this 😂😂
